Main/Controllers/ConfigurationController.cs

```diff
@@ -44,5 +45,15 @@ public IActionResult UpdateConfiguration([FromBody] CollectionConfiguration newC
       _configurationProvider.MaxElementsInCollection = newConfig.MaxElementsInCollection;
       return Ok();
     }
+
+    [Authorize]
+    [HttpGet("spotifyclientid")]
+    public ActionResult<string> GetSpotifyClientId()
+    {
+      var entry = _dbContext.Configuration.SingleOrDefault(c => c.Key == "SpotifyClientId");
```

Comment:
It is fine for our use case, but in real life it really should be done with secrets:
https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.1&tabs=windows

Reply:
Didn't knew about it